<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employeewindow</class>
 <widget class="QDialog" name="employeewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="empID">
              <property name="text">
               <string>EmployeeID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="empName">
              <property name="text">
               <string>Employee Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="empCost">
              <property name="text">
               <string>Cost per hour</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="empIdInput"/>
            </item>
            <item>
             <widget class="QLineEdit" name="empNameInput"/>
            </item>
            <item>
             <widget class="QLineEdit" name="empCostInput"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="addEmpbtn">
              <property name="text">
               <string>Add Employee</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteEmpbtn">
              <property name="text">
               <string>Delete Employee</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableWidget" name="emptableWidget">
         <property name="columnCount">
          <number>3</number>
         </property>
         <column>
          <property name="text">
           <string>employeeID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Employee_Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cost_per_hour</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="showEmpData">
           <property name="text">
            <string>Load All Employee</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="empClose">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
